src/components/builder-footer/builder-footer.js

```diff
@@ -14,7 +14,7 @@ const BuilderFooter = ({SideBarAndIsHome, HomePage}) => {
       ${SideBarAndIsHome ? builderFooterStyles.home : ''}
       `}
     >
-      <span>2019® - design by <a href="http://www.mybitdesign.com" target="_blank" rel="noreferrer noopener">www.mybitdesign.com</a></span>
+      <span>2019® - design by <a href="http://www.mybitdesign.com" target="_blank" rel="noreferrer noopener">mybitdesign</a> - built by <a href="http://datavaluepeople.com" target="_blank" rel="noreferrer noopener">datavaluepeople</a></span>
```

Comment:
I think the link to datavaluepeople.com is not needed. 

I would remove the `<a>` tag and just have datavaluepeople. This will remove the link and the underline styling. Which I think is correct.

Reply:
I put it in for:
a) then it looks matching with the styling for `mybitdesign`
b) self-referential stuff is classic computer vibes (e.g. GNU meaning "GNU is Not Unix") so thought it was funny ;)

Reply:
Alright, then I would add it as a gatsby link to home. You can see gatsby links in the file: src/components/side-bar/side-bar.js (`<Link>`) and to link to home you would need `to='/'`.